lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -132,6 +132,12 @@ tokenize([], EndLine, _Column, #elixir_tokenizer{terminators=[{Start, {StartLine
   Message = io_lib:format("missing terminator: ~ts (for \"~ts\" starting at line ~B)", [End, Start, StartLine]),
   {error, {EndLine, Message, []}, [], Tokens};
 
+% ## VC merge conflict
```

Comment:
That's the thing: file has mixed style.
I think we should go with single `%` here as in the style of the next comment `% Base integers`,
and fix overall style separately (in the another PR I suppose).
_Even `% %%` exists there._ :frowning: